nixos/modules/services/x11/xautolock.nix

```diff
@@ -31,7 +31,17 @@ in
           type = types.string;
 
           description = ''
-            The script to use when locking the computer.
+            The script to use when automatically locking the computer.
+          '';
+        };
+
+        nowlocker = mkOption {
+          default = null;
+          example = "i3lock -i /path/to/img";
+          type = types.nullOr types.string;
+
+          description = ''
+            The script to use when manually locking the computer with `xautolock -locknow`.
```

Comment:
I believe
```
`xautolock -locknow`
```
here should be
```
<command>xautolock -locknow</command>
```
since the description is interpreted as docbook code.